package.json

```diff
@@ -10,13 +10,18 @@
     "start": "nf start",
     "lint": "eslint --color *.js; exit 0;",
     "test:watch": "mocha -w",
-    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec"
+    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec",
+    "migrate": "sequelize db:migrate",
+    "migrate:undo:all": "sequelize db:migrate:undo:all",
+    "migrate:test": "sequelize db:migrate --url "
```

Comment:
I did not know until reading the notes again that this requires an additional parameter - is there a way to obtain this from `process.env`?